example.env.bt

```diff
@@ -2,4 +2,8 @@ TRACING_STRATEGY=bt
 POST_DIAGNOSIS_KEYS_URL=gaen_server_url_here
 DOWNLOAD_KEY_FILE_URL=gaen_server_url_here
 INDEX_FILE_URL=gaen_server_url_here
-AUTHORITIES_YAML_ROUTE=yaml_route_here
\ No newline at end of file
+HMAC_KEY=abcd1234
+GAEN_VERIFY_URL=https://example.com
+GAEN_VERIFY_API_TOKEN=abcd1234
```

Comment:
mind making a PR to `pathcheck-mobile-resources` to update the `.env`s with the production values? 😁